### ISSUE TITLE
Resource generator doesn't support "nested" values

### ISSUE THREAD

**Issue by markbates** [gobuffalo/buffalo#364]
```
buffalo g resource admin/planes name
Buffalo version 0.8.1.dev

--> actions/admin/planes.go
--> actions/admin/planes_test.go
--> locales/admin/planes.en-us.yaml
--> templates/admin/planes/_form.html
--> templates/admin/planes/edit.html
--> templates/admin/planes/index.html
--> templates/admin/planes/new.html
--> templates/admin/planes/show.html
--> buffalo db g model admin/plane name
v3.13.7

--> models/admin/plane.go
--> models/admin/plane_test.go
--> goimports -w .
actions/admin/planes.go:22:11: expected type, found '/'
actions/admin/planes.go:38:2: expected declaration, found 'if'
actions/admin/planes_test.go:9:34: expected '(', found '/'
actions/admin/planes_test.go:10:2: missing ',' in parameter list
actions/admin/planes_test.go:11:1: expected 'IDENT', found '}'
actions/admin/planes_test.go:13:1: missing ',' in parameter list
actions/admin/planes_test.go:17:34: expected '(', found '/'
actions/admin/planes_test.go:18:2: missing ',' in parameter list
actions/admin/planes_test.go:19:1: expected 'IDENT', found '}'
actions/admin/planes_test.go:21:1: missing ',' in parameter list
actions/admin/planes_test.go:25:34: expected '(', found '/'
actions/admin/planes_test.go:26:2: missing ',' in parameter list
actions/admin/planes_test.go:27:1: expected 'IDENT', found '}'
models/admin/plane.go:12:11: expected type, found '/'
models/admin/plane.go:22:2: expected declaration, found 'return'
models/admin/plane.go:26:11: expected type, found '/'
models/admin/plane.go:31:2: expected declaration, found 'return'
models/admin/plane_test.go:5:16: expected '(', found '/'
```

**Comment by markbates** [comment]
I'm revisiting this. I've closed the ticket for #394 which directly impacted this ticket. I've had a change of heart as to how this ticket should be implemented. Here is my take on how this should work now:

```bash
$ buffalo g resource admin/plane

--> actions/admin_planes.go
--> actions/admin_planes_test.go
--> locales/admin/planes.en-us.yaml
--> templates/admin/planes/_form.html
--> templates/admin/planes/edit.html
--> templates/admin/planes/index.html
--> templates/admin/planes/new.html
--> templates/admin/planes/show.html
--> buffalo db g model plane name
v3.13.7

--> models/plane.go
--> models/plane_test.go
--> goimports -w .
```

```bash
$ buffalo g resource admin/plane --model-name=AdminPlane

--> actions/admin_planes.go
--> actions/admin_planes_test.go
--> locales/admin/planes.en-us.yaml
--> templates/admin/planes/_form.html
--> templates/admin/planes/edit.html
--> templates/admin/planes/index.html
--> templates/admin/planes/new.html
--> templates/admin/planes/show.html
--> buffalo db g model admin_plane name
v3.13.7

--> models/admin_plane.go
--> models/admin_plane_test.go
--> goimports -w .
```

**Comment by markbates** [comment]
@apaganobeleno think you crank this out in the next two weeks given the new requirements?

**Comment by apaganobeleno** [comment]
@markbates yes, i'll work on this over next 2 weeks

**Comment by markbates** [comment]
awesome! :)

**Comment by apaganobeleno** [comment]
@markbates i'm assuming for the router path should be `/admin/planes`, am i right ?

**Comment by markbates** [comment]
That would be correct. :)

**Comment by apaganobeleno** [comment]
@markbates aha! this one is tricky! that --model-name flag was not there :D, going for it

**Comment by apaganobeleno** [comment]
Closed by #469
